datafusion/core/tests/expr_api/mod.rs

```diff
@@ -28,6 +28,23 @@ use std::sync::{Arc, OnceLock};
 
 mod simplification;
 
+#[test]
+fn test_octet_length() {
```

Comment:
can we move it to slt?

Reply:
Since this fix is for the expr_fn API I don't think we can move this to .slt (the code path invoked needs to be invoked programatically, not via SQL)